includes/classes/mapping-post.php

```diff
@@ -76,31 +77,32 @@ public static function get( $post, $throw_error = false ) {
 			if ( $throw_error ) {
 				throw $e;
 			}
+
 			return false;
 		}
 	}
 
 	/**
 	 * Get the full post object.
 	 *
-	 * @since  3.0.0
-	 *
-	 * @param  mixed $post Post id or object.
+	 * @param mixed $post Post id or object.
 	 *
+	 * @return WP_Post Post object.
 	 * @throws Mapping_Post_Exception If post could not be retrieved.
 	 *
-	 * @return WP_Post Post object.
+	 * @since  3.0.0
+	 *
 	 */
 	protected static function get_post( $post ) {
 
 		$post = $post instanceof WP_Post ? $post : get_post( $post );
 
 		if ( ! $post ) {
-			throw new Mapping_Post_Exception( __CLASS__ . ' expects a WP_Post object or post ID.', __LINE__, $post );
+			throw new Mapping_Post_Exception( __CLASS__ . ' expects a WP_Post object or post ID.', __LINE__, wp_kses_post( print_r( $post, true ) ) );
```

Comment:
Why do we `print_r` as wouldn't that output to the frontend?

Reply:
Good spot! My thinking was it was just for reporting as we don't catch the exception anywhere, but, I don't know if I fully trust that it ins't being caught anywhere 😂 

I've whacked the ignore warning comment above to allow it!